src/main/java/uk/gov/justice/digital/hmpps/keyworker/dto/KeyworkerDto.java

```diff
@@ -60,4 +62,11 @@
 
     @ApiModelProperty(value = "Key worker is eligible for auto allocation.")
     private Boolean autoAllocationAllowed;
+
+    /**
+     * @return fullname (last name first name in lower case)
+     */
+    public String getFullName() {
+        return StringUtils.lowerCase(StringUtils.join(Arrays.asList(lastName, firstName), " "));
+    }
 }
```

Comment:
is this returned as part of the json?  Should it be formatted properly or excluded?

Reply:
We could put an ignore on it.  Perhaps I should move it to a private method.  ....

Reply:
@lbennett-moj I've push a new change as a static method